dolphinscheduler-master/src/main/java/org/apache/dolphinscheduler/server/master/runner/WorkflowExecuteThreadPool.java

```diff
@@ -182,6 +185,18 @@ private void notifyProcessChanged(ProcessInstance finishProcessInstance) {
         }
     }
 
+    private void crossWorkflowParameterPassing(ProcessInstance finishProcessInstance, TaskInstance taskInstance) {
+        try {
+            MasterTaskExecuteRunnable masterTaskExecuteRunnable =
+                    MasterTaskExecuteRunnableHolder.getMasterTaskExecuteRunnable(taskInstance.getId());
+            masterTaskExecuteRunnable.getILogicTask().getTaskParameters()
+                    .setVarPool(finishProcessInstance.getVarPool());
+            log.info("Cross workflow parameter passing success");
```

Comment:
```suggestion
            log.info("Cross workflow parameter passing success, processInstanceId: {}, taskInstanceId");
```

It's best to add more effective information for troubleshooting purposes.